Python/optimizer.c

```diff
@@ -1127,11 +1245,19 @@ _Py_ExecutorInit(_PyExecutorObject *executor, _PyBloomFilter *dependency_set)
 void
 _Py_ExecutorClear(_PyExecutorObject *executor)
 {
+    if (!executor->vm_data.valid) {
+        return;
+    }
     unlink_executor(executor);
     PyCodeObject *code = executor->vm_data.code;
     if (code == NULL) {
         return;
     }
+    for (uint32_t i = 0; i < executor->exit_count; i++) {
+        Py_DECREF(executor->exits[i].executor);
+        executor->exits[i].executor = &COLD_EXITS[i];
+        executor->exits[i].temperature = INT16_MIN;
```

Comment:
It's the smallest value possible:
* It is unlikely that any other code would set it to that value, so it might help debugging.
* It will below any threshold, so even if the executor is still running, it will exit safely

In theory we shouldn't need to set it at all, but this seems safer.